applications/rag/main.tf

```diff
@@ -131,7 +131,9 @@ module "cloudsql" {
   instance_name = var.cloudsql_instance
   namespace     = var.kubernetes_namespace
   region        = local.cloudsql_instance_region
-  depends_on    = [module.namespace]
+  network_name  = var.network_name
+  // Depend on module.infra to ensure network is created
+  depends_on = [module.namespace, module.infra]
```

Comment:
Not required.  as kubernetes provider will be configured after finishing the infra module.. so implicitly it depends on already.

Reply:
Removed